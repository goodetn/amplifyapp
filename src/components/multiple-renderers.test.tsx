import React from 'react';
import { Link as RRDLink } from 'react-router-dom';
import { create } from 'react-test-renderer';

type LinkProps = {
  page: string;
  children?: React.ReactNode;
}
const Link = (props: LinkProps) => <RRDLink to={props.page}>{props.children}</RRDLink>

it('renders correctly', () => {
  const tree = create(<Link page="http://www.facebook.com">Facebook</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

// https://jestjs.io/docs/snapshot-testing#snapshot-testing-with-jest     =>  snapshot obsolete.
