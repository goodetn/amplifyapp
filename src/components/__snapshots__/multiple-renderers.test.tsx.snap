// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo512.png"
    />
    <div>
      <address>
        Contact 
        tot
         via
         
        <a
          data-testid="email"
          href="mailto:tottranrotate@gmail.com"
        >
          email
        </a>
        or on their 
        <a
          data-testid="site"
          href="toithietke.vn"
        >
          website
        </a>
        .
      </address>
    </div>
  </header>
</div>
`;
